src/pages/Basket.js

```diff
@@ -1,5 +1,10 @@
+import ProductList from "../Components/ProductList";
+import products from "../models/dummydata.json"
 function BasketPage() {
-    return <div></div>
+    return (
+        <> <ProductList 
+        items={products}/> </>
```

Comment:
Why is there a {products} prop here linking to the json data if you have already got props assigned to Product List earlier with multiple props?